web/e2e/config/configs.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+
+export type SeleniumCapabilitiesType = {|
+  browserName: string,
+  browserVersion: string,
+  platformName?: string,
+  chromeOptions?: {| args: Array<string> |}
+|}
+
+export type BrowserstackCapabilitiesType = {|
+  os: string,
+  os_version: string,
+  browserName: string,
+  browser_version: string,
+  'browserstack.local': boolean,
+  'browserstack.user': string,
+  'browserstack.key': string,
+  'browserstack.debug': boolean,
+  project: string
+|}
+
+export type EndToEndConfigType = {|
+  url: string,
+  prefix: string,
+  browserstack?: boolean,
+  caps: SeleniumCapabilitiesType | BrowserstackCapabilitiesType
+|}
+
+/*
+  LOCAL
+ */
+
+exports.local_chrome = ({
+  url: 'http://localhost:4444/wd/hub',
+  prefix: 'IG LOCAL',
+  caps: {
+    browserName: 'chrome', // selenium has lowercased browsernames
+    browserVersion: '88'
```

Comment:
Probably should be removed as we probably do not want to test against different versions of chrome